index.html

```diff
@@ -121,13 +136,34 @@ <h5 class="card-title">Breath-test gadgets tell you what to eat</h5>
             <span class="sr-only">Next</span>
           </a>
         </div>    
-      
-      
 
+    <!--Footer Jumbotron-->    
+    <footer>
+      <div class="jumbotron jumbotron-fluid">
+        <div class="container">
+          <div class="row">
+            <div class="col-md-4 offset-4">
+              <a class="nav-link" href="#"><i class="fab fa-instagram"></i></a>
```

Comment:
It would help if we'll put <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"> for them in the header :)